src/tests/5e-SRD-Alignments.json

```diff
@@ -0,0 +1,64 @@
+[
+    {
+        "index": "lg",
```

Comment:
I can't say I'm a big fan of using the abbreviation for the `index` and `name`, here. I feel like supplying the abbreviation as a separate property might be more suitable?

```json
{
  "index": "lawful-good",
   "name": "Lawful Good",
   "abbreviation": "LG",
   "desc": "Lawful good (LG) creatures can be counted on to do the right thing as expected by society. Gold dragons, paladins, and most dwarves are lawful good.",
   "url": "/api/alignments/lawful-good"
}
```